lib/application/personal_information/personal_information_bloc.dart

```diff
@@ -0,0 +1,25 @@
+// ignore_for_file: invalid_use_of_visible_for_testing_member
+
+import 'package:bloc/bloc.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:injectable/injectable.dart';
+
+import '../../domain/profile/user_profile.dart';
+import '../../domain/user/i_profile_repository.dart';
+
+part 'personal_information_event.dart';
+part 'personal_information_state.dart';
+part 'personal_information_bloc.freezed.dart';
+
+@injectable
+class PersonalInformationBloc
+    extends Bloc<PersonalInformationEvent, PersonalInformationState> {
```

Comment:
Could use a cubit instead since it doesn't really need an event.

Reply:
We don't really use cubits, maybe we can create an issue to convert BLoCs to Cubit were possible later.